### ISSUE TITLE
Deep neural network prediction of genome-wide transcriptome signatures – beyond the Black-box

### ISSUE THREAD

**Issue by cmclean5** [cmclean5/PublicHealthModels#11]
## TL;DR

Paper uses a light-up technique to inspect the trained neural network to ask whether training a DNN on gene expression data could learn a predictive transcription factor TF network human transcriptomic profiles.

### Paper Link

https://www.nature.com/articles/s41540-022-00218-9

### Author/Institution

Magnusson, R (Linköping University) Tegnér, J.N (Abdullah University of Science and Technology, Karolinska Institutet), Gustafsson, M (Linköping University)

## Cite

Magnusson, R., Tegnér, J.N. & Gustafsson, M. Deep neural network prediction of genome-wide transcriptome signatures – beyond the Black-box. npj Syst Biol Appl 8, 9 (2022). https://doi.org/10.1038/s41540-022-00218-9

## Comments

Code used to generate the results, the DNN with 250 hidden nodes in two layers, can be found at: https://github.com/rasma774/tf_target_white_box_dl
